### ISSUE TITLE
FYPO:0006518 & FYPO:0006660 difference

### ISSUE THREAD

**Issue by Antonialock** [pombase/fypo#3485]
In 30116786 Val used FYPO:0006660  loss of viability upon G0 to G1 transition

I am thinking about the difference between this and 
FYPO:0006518 loss of viability in G0 

Because usually "viability in G0" is assessed by regrowing cells (so you don't know if they lose viability when trying to exit or when maintaining)?

in 19366728 I have a number of mutants that have frgamented DNA in G0 (so these are clearly dead in maintenance).

what do I do with the rest? annotate to FYPO:0006660 a la Val's session or FYPO:0006518? I don't know if it is maintenance or transition that kills them.

**Comment by ValWood** [comment]
So we have 
a) Some cell s which clearly cannot maintain  G0
b) Some cells which can maintain G0 but cannot exit G0 (mitotic competence)
c) Some cells where we do not know the exact issue

Is that accurate? In this case do we need a grouping term to cover a and b?

**Comment by Antonialock** [comment]
Yes I think so.

How did they establish " loss of viability upon G0 to G1 transition"  in 30116786 ?

**Comment by ValWood** [comment]
I don't remember but there was some correspondence about it, I will try to locate....

**Comment by ValWood** [comment]
not described in ticket anyway
https://github.com/pombase/fypo/issues/3346

**Comment by mah11** [comment]
As Val says, in principle the difference is clear enough: if cells die while in G0, use FYPO:0006518, whereas if they remain quiescent but alive until they try to start up the cycle in G1 and only then die, use FYPO:0006660. I'm a little unclear on how to tell the difference in practice - is there some way to see if G0 cells are alive other than to see if they form colonies (which requires going through the G0-to-G1 transition)? I guess that's basically the same question as in Antonia's last comment.

Also, at present FYPO:0006518 and FYPO:0006660 are siblings, but the way FYPO:0006518 is defined makes it a bit ambiguous whether it's "really" the general case (Val's 'c') or the G0-maintenance-specific case (Val's 'a'). FYPO:0006660 is unmistakably case 'b'. I can adjust FYPO:0006518 in either direction. Which is better, make it generic or make it the maintenance term?

Finally, what term names would make the 'a' vs. 'c' distinction clear?

**Comment by Antonialock** [comment]
hmm how about

- loss of viability in G0 (at some stage in G0, to encompass both maintenance and exit). this would be the parent term for any case where cells die after entering G0, but we dont know if it's because they die when they try to exit, or they die whilst maintaining G0
  - FYPO:0006660 loss of viability upon G0 to G1 transition - I guess here they are alive in G0 but die in the transition (@ValWood  how do you make this distinction?)
  - loss of viability during maintenance of G0 - in the paper I was looking at I used 'fragmented DNA' as a proxy for this - cells cannot be alive without intact DNA. Other ways to measure this (that I can imagine) would be metabolic turnover

**Comment by ValWood** [comment]
They are alive in G0 but cannot grow vegetatively after they have been in G0 (although they were fine vegetatively before). This therefore implies they have a problem getting  *out* of G0" which must be the transition.

**Comment by mah11** [comment]
OK, that helps. But for me there's still the question of how we know they're alive in G0 - that may be the key to capturing the distinctions in phenotype term defs and comments.

**Comment by Antonialock** [comment]
Yes, what marker would you use to state that they are alive in G0 but can’t exit?

**Comment by ValWood** [comment]
I can't remember exactly I would need to go back to the paper. Don't they show that they are metabolically active IIRC? It's so long ago...

**Comment by ValWood** [comment]
quantitative metabolomics analysis show the cells are cycling in G0

adding back nitrogen and they can't re-enter vegetative growth.

Therefore they must be  G0->G1 transition defective?

**Comment by ValWood** [comment]
Actually I'm not sure that a defect in the transition is the best way to describe the "mitotic competence" phenotype.

Now I looked at this it seems that   mitotic competence is
" the ability to  sustain G0 + the ability to reenter mitosis"
I don't know if this is really a transition capability.... but it might be. I remember being confused by this at the time.

It's really  the group of processes that must be present in G0 in order to reenter mitosis.

**Comment by ValWood** [comment]
Ignore I was trying to delete the comment I was writing the term for G0-G1 transition does seem to be the correct way to describe this phenotype. We are not saying anything about the processes causing the defect, only that a block is the observable phenotype.
